cypress/integration/TokenGovernancePage/TokenGovernancePage.ts

```diff
@@ -0,0 +1,35 @@
+import {Given, Then} from "cypress-cucumber-preprocessor/steps";
+import {TokenGovernancePage} from "../../pageObjects/pages/TokenGovernancePage";
+import {LandingPage} from "../../pageObjects/pages/LandingPage";
+
+Given(`User has opened the {string} page`, (page) => {
+  LandingPage.openPage(page)
+});
+
+Then("User switches network for {string} and validates data", (network) => {
+  TokenGovernancePage.switchLNetworkAndValidateTokens(network)
```

Comment:
"switchLNetwork" is the "L" a typo or it means something?

Reply:
👍